### PR TITLE
add: solutions to prime reversion & vaporcode

### PR DESCRIPTION
**Pull request by aldotele**

### CODE REVIEW

**Comment by micheleriva** [comment]
@aldotele awesome solution, it works perfectly. I just left one little comment to make this code 10/10 🙂

**Comment by aldotele** [comment]
@micheleriva thanks! Do you mean that you left a little comment within the code? I cannot see that.

**Comment by micheleriva** [comment]
@aldotele you should see the comment here: https://github.com/tomorrowdevs-projects/challenges/pull/11/files/8298323b2ef1dcfbd919567fd66ff6a955bdbcb6#diff-46deb0cb1a014933035aed9ab1047d2a522a1193942469343751e278332d01f6

**Comment by aldotele** [comment]
@micheleriva still not able to see it unfortunately

File: challenges/intermediate/prime-reversion/solutions/aldotele/main.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+def find_primes_in_list(numbers):
+    """
+    :param numbers: list of integers
+    :return: a list of prime integers
+    """
+    primes = []
+    for num in numbers:
+        if num > 1:
+            for i in range(2,num):
```
[the remark this answers is not in the document]
Comment (micheleriva) [reply challenges/intermediate/prime-reversion/solutions/aldotele/main.py:23]: @aleattene please remove duplicated functions

**Pull request merged**